src/NServiceBus.Core/Performance/Metrics/MetricsExtensions.cs

```diff
@@ -0,0 +1,52 @@
+namespace NServiceBus.Performance.Metrics
+{
+    using System;
+    using System.Collections.Generic;
+    using Features;
+
+    /// <summary>
+    /// Extension methods to configure OpenTelemetry metrics.
+    /// </summary>
+    public static class MetricsExtensions
+    {
+        /// <summary>
+        /// Configures NServiceBus to report messaging metrics using OpenTelemetry.
+        /// </summary>
+        /// <param name="config">The <see cref="EndpointConfiguration" /> instance to apply the settings to.</param>
+        public static EndpointConfiguration EnableOpenTelemetryMetrics(this EndpointConfiguration config)
```

Comment:
This should have a `void` return type. To be consistent with other config methods we would only return something if further config is needed. i.e. We could return an `OpenTelemetryMetricsConfiguration`.